pkg/ccl/sqlccl/csv.go

```diff
@@ -686,7 +686,7 @@ func importPlanHook(
 			// TODO(dt): verify db exists
 		}
 
-		comma := ','
+		var comma rune
```

Comment:
This is just so the default is only in one place, yeah? (As in, this was technically correct before, right?) Just checking my own understanding.